sensors/ev3_uart_sensor_ld.c

```diff
@@ -66,7 +66,8 @@
 #endif
 
 #define EV3_UART_MAX_DATA_SIZE		32
-#define EV3_UART_MAX_MESSAGE_SIZE	(EV3_UART_MAX_DATA_SIZE + 2)
+// extra bytes for: main command byte, INFO command byte, final checksum
```

Comment:
Could you please change this to `/* */` style comments (it is the Linux kernel style.